cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -1514,7 +1515,7 @@ def get_error_code(self):
 
 	## Returns the graphstring.
 	def GetGraphstring(self):
-		if type(self.parameter_name) is types.StringType:
+		if type(self.parameter_name) is bytes:
```

Comment:
This might be exactly what id not desired here, we probably really want a `str` I think